README.md

```diff
@@ -1,8 +1,17 @@
-# transport_drivers
-A set of ROS2 drivers for transport-layer protocols. Currently utilizes Boost::ASIO for most transport-layer functionality.
+## Supported Drivers:
 
-## udp_driver
-A package which contains a templated C++ class (which inherits from `rclcpp::Node`) which encapsulates basic receiving (and soon sending) of UDP network data.
-
-## serial_driver
-A package which contains a templated C++ class (which inherits from `rclcpp::Node`) which encapsulates basic receiving (and soon sending) of serial data.
+* **UDP Driver**
+    * Tracking PR: https://github.com/ros-drivers/transport_drivers/pull/31
+    * Implemented `IoContext` (The main I/O Context that can be configured as single or multi thread with async task execution support)
+    * Implemented `UdpSocket` (The UDP socket functionality that working in blocking and non-blocking (Async with task execution) modes)
+    * Implemented `UdpDriver` Facade (Facade design pattern implementation that acts as a sender and receiver UDP socket pair container)
+    * Implemented `ROS2` and `raw` buffer message converters for Int, UInt and Float variants in std_msgs namespace.
+    * Implemented `UdpDriverNode` example and its test.
+    * Implemented IoContext, UdpSocket and UdpDriver tests.
+  
+* **TCP Driver**
+    * Not Implemented yet.
+    
+* **Serial Driver**
+    * Planning to implement with new `IoContext` (Blocking and NonBlocking) functionality.
+    * Current implementation contains legacy driver code (A package which contains a templated C++ class (which inherits from `rclcpp::Node`) which encapsulates basic receiving (and soon sending) of serial data.)
```

Comment:
I'm not fond of deprecating the existing implementation with just an IOU for a new implementation. Please create classes with common functionality instead of moving the UDP version to the base. We want to keep the UDP driver and the Serial driver in separate packages.

Reply:
Since there is no provided reason behind two separate packages, I will move UDP to its package and leave serial package as before.

Reply:
No one deprecated Serial port. I was planned for implementing Serial port socket like the UDP one after finalizing UDP.
After that we can deprecate legacy code.

Since there is no provided reason behind two separate packages, But its OK, I will move UDP to its own package and leave serial package as before.

Reply:
Done 78c1d1c6925d6fbfa505c690b349c70f33e2b2ed.